elements/pfe-tabs/src/pfe-tab-panel.scss

```diff
@@ -3,11 +3,19 @@
 $LOCAL: tabs;
 
 $LOCAL-VARIABLES: (
-  focus:  pfe-var(link)
+
+  focus:  pfe-var(link),
+  panel: ( // to avoid collisions with tab header
```

Comment:
```suggestion
  panel: ( // to avoid collisions with tab header
```

That's not really a risk here because it's scoped to the pfe-tab-panel component.

Reply:
but if someone overrides a variable at the root level, it will cascade down to both the tab header & panel right?

Reply:
Right

Reply:
The padding on tab headers is very different than the panels, so i think it makes sense to have independent vars for each:
![image](https://user-images.githubusercontent.com/456863/88709119-b10fa400-d0e2-11ea-999d-ddb81fd2a445.png)